src/services/env/Env.ts

```diff
@@ -65,7 +66,7 @@ export default class Env {
 
     // Falls back to a sensible default when encountering a malformed JSON payload
     // or non-replaced template.
-    return PATH_CONFIG_DEFAULT
+    return getPathConfigDefault(import.meta.env.PROD ? window.location.origin : import.meta.env.VITE_KUMA_API_SERVER_URL)
```

Comment:
Suggestion: Could this if be moved out into the vite config by having `getPathConfigDefault` take a lambda: `(): string {}` seems like it could help keeping the complexity in a single place or do a systematic if `VITE_KUMA_API_SERVER_URL || window.location.origin`?